library/rtsp/src/test/java/com/google/android/exoplayer2/source/rtsp/RtspClientTest.java

```diff
@@ -113,6 +190,7 @@ public void onSessionTimelineRequestFailed(
             EMPTY_PLAYBACK_LISTENER,
             /* userAgent= */ "ExoPlayer:RtspClientTest",
             RtspTestUtils.getTestUri(rtspServer.startAndGetPortNumber()),
+            /* socketFactory */ null,
```

Comment:
since `RtspMediaSource.Factory` will never use a null value, should this be `SocketFactory.getDefault()`?

Reply:
I agree. :-)